ftw/file/upgrades/20150504090723_enable_tiny_mce_image_upload/tinymce.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object>
+ <resourcetypes>
+  <imageobjects purge="True">
+    <element value="File"/>
```

Comment:
:+1: agreed
@shylux please remove the tinymce configuration (upgradestep + profile) and instead add a note to the readme that ftw.file supports beeing an image and how tinymce can be configured.
you may revert the upgrade related changes..